srcpkgs/rtags/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'rtags'
 pkgname=rtags
-version=2.16
-revision=3
+version=2.18
+revision=1
 nocross=yes
-wrksrc="${pkgname}"
+wrksrc="$pkgname"
 build_style=cmake
+# Don't run tests in do_install, please.
+configure_args="-DSKIP_CTEST=TRUE"
 make_build_args="VERBOSE=1"
 makedepends="pkg-config git clang llvm libressl-devel zlib-devel"
```

Comment:
You could also clean this up since you are at this package and put `pkg-config git clang` into host makedepends.